aiidalab_widgets_base/viewers.py

```diff
@@ -217,6 +219,23 @@ def change_camera(change):
 
         super().__init__(children, **kwargs)
 
+    def _camera_selection(self, default_camera):
```

Comment:
If that works for you, I would propose to make it just another tab and call it `_camera_tab(self, ..)`